edx-platform/pearson-vue-theme/lms/templates/index.html

```diff
@@ -0,0 +1,58 @@
+<%page expression_filter="h"/>
+<%inherit file="main.html" />
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.urls import reverse
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<div class="background-login"></div>
+
+<div class="login-container">
+  <div>
+    <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
+  </div>
+  <div class="signin-container">
+    <div class="toggle-form">
+      <h1>${_("Welcome!")}</h1>
+      <br>
+      <span class="text">${_("Sign in or create an account to access your courseware.")}</span>
+    </div>
+    <div class="login-providers">
+      <a href="/login">
+        <button class="button-vue">
+          <span aria-hidden="true">${_("Continue to sign in")}</span>
+          <span class="icon fa fa-arrow-right"></span>
+        </button>
+      </a>
+    </div>
+  </div>
+</div>
+
+% if show_homepage_promo_video:
+  <section id="video-modal" class="modal home-page-video-modal video-modal">
+    <div class="inner-wrapper">
+      <iframe title="YouTube Video" width="640" height="360" src="//www.youtube.com/embed/${homepage_promo_video_youtube_id}?showinfo=0" frameborder="0" allowfullscreen></iframe>
+    </div>
+  </section>
+% endif
```

Comment:
Is this block being used? @luismorenolopera

Reply:
no, this is something that comes in the original template.  Should I remove everything that is not used from the original template?

Reply:
If it's not being used, I think it's fine to remove it. @luismorenolopera

Reply:
I have removed it along with other lines that are not in use @Squirrel18